pkg/runtime/types.go

```diff
@@ -1,16 +1,46 @@
 package runtime
 
-import "io"
+import (
+	"io"
+	"time"
+
+	meta "github.com/weaveworks/ignite/pkg/apis/meta/v1alpha1"
+)
 
 type ImageInspectResult struct {
 	ID          string
 	RepoDigests []string
 	Size        int64
 }
 
+type Bind struct {
+	HostPath      string
+	ContainerPath string
+}
+
+type ContainerConfig struct {
+	Cmd          []string
+	Hostname     string
+	Labels       map[string]string
+	Binds        []*Bind
+	CapAdds      []string
+	Devices      []string
+	StopTimeout  uint32
+	AutoRemove   bool
+	NetworkMode  string
+	PortBindings meta.PortMappings
+}
+
 type Interface interface {
 	InspectImage(image string) (*ImageInspectResult, error)
 	PullImage(image string) (io.ReadCloser, error)
+	ExportImage(image string) (io.ReadCloser, string, error)
 	GetNetNS(containerID string) (string, error)
```

Comment:
rename this to `GetContainerNetns()` and move to the container section?

Reply:
Done in 6e60eba.